src/wallet-web-api-swagger/Main.hs

```diff
@@ -43,7 +45,18 @@ import           Pos.Wallet.Web             (CAddress, CCoin, CCurrency,
                                              CUpdateInfo, CWallet, CWalletAssurance,
                                              CWalletInit, CWalletMeta, CWalletRedeem,
                                              CWalletType, SyncProgress, WalletError,
-                                             walletApi)
+                                             walletApi,
+                                             TestReset, GetWallet, GetWallets,
```

Comment:
It'd be better to import `Pos.Wallet.Web` qualified and avoid this huge import list.